lib/Backend/SccLiveness.cpp

```diff
@@ -705,7 +705,7 @@ SCCLiveness::ExtendLifetime(Lifetime *lifetime, IR::Instr *instr)
             }
             NEXT_SLISTBASE_ENTRY
         }
-        AssertMsg(lifetime->end > instr->GetNumber(), "Lifetime end not set correctly");
+        AssertMsg(lifetime->end >= instr->GetNumber(), "Lifetime end not set correctly");
```

Comment:
We should have 2 asserts here one for ARM64 with >= and other > because only ARM64 can have src on label end.